tests/test_main_run_main.py

```
@@ -0,0 +1,471 @@
+    _mock_run_with_exception,
+    _mock_run_with_keyboard_interrupt,
+    mock_config,
+    mock_config_edge_cases,
+)
+
+# =============================================================================
+# Banner tests
+# =============================================================================
+
+
+def test_print_banner(mock_config):
```

Ruff (0.15.12)

[warning] Unused function argument: `mock_config`

(ARG001)